calculate_coverage_stub.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Check if coverage/lcov.info exists and print basic stats
+if [ -f coverage/lcov.info ]; then
+  TOTAL_LINES=$(grep -c "^DA:" coverage/lcov.info)
+  COVERED_LINES=$(grep "^DA:" coverage/lcov.info | grep -v ",0$" | wc -l)
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)